src/routes/Dashboard/SearchPlateForm.jsx

```diff
@@ -28,6 +30,15 @@ export default function SearchPlateForm({onSearch, onClear}) {
         }
     }, [inputValue, vehicleId, resetForm])
 
+    useEffect(() => {
+        if (searchParams.has('plate')) {
+            const vehiclePlate = searchParams.get('plate')
+            if (vehiclePlate.length === 6) {
+                setInputValue(`${vehiclePlate.substring(0, 3)} ${vehiclePlate.substring(3, 6)}`)
```

Comment:
This needs to be a reusable function that is used here and when the input changes